src/Http.ts

```diff
@@ -0,0 +1,33 @@
+import Axios, { AxiosInstance } from 'axios'
+
+export default class Http {
+  public host: string;
+  public axios: AxiosInstance
+
+  constructor() {
+    this.host = process.env.SPREE_HOST || 'http://localhost:3000/api/v2/storefront'
```

Comment:
Maybe merge custom host PR with this PR? Because old PR will be invalid after merging this.